content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: "CNI version-related breaking change in Kubernetes 1.24 GA"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+canonicalUrl: https://kubernetes.dev/blog/2022/04/28/cni-version-related-breaking-change-k8s-1.24/
+---
+
+**Authors & Contributors:** [Mike Brown](https://github.com/mikebrow), [Rolfe Dlugy-Hegwer](https://github.com/rolfedh),[Nate W.](https://github.com/nate-double-u)
+
+---
+
+**IMPORTANT** Urgent upgrade notes
```

Comment:
I'm not sure the release note structure works here as we're writing a blog entry.

I think here we should highlight the nature of the issue rather than that it is an upgrade note.

Reply:
Updated. Thanks.